actions.go

```diff
@@ -156,25 +199,35 @@ func (c *Action) Fatalf(msg string, args ...interface{}) {
 // Warningf prints a warning-level message. The arguments follow the standard
 // Printf arguments.
 func (c *Action) Warningf(msg string, args ...interface{}) {
-	fmt.Fprintf(c.w, warningFmt, c.fields, fmt.Sprintf(msg, args...))
+	// ::warning <c.fields>::<msg, args>
+	c.IssueCommand(&Command{
+		Name:       warningCmd,
+		Message:    fmt.Sprintf(msg, args...),
+		Properties: c.fields,
+	})
 }
 
 // WithFieldsSlice includes the provided fields in log output. "f" must be a
 // slice of k=v pairs. The given slice will be sorted.
```

Comment:
I'm gonna merge this, but can we document the panic here?

Reply:
@ashutoshgngwr from https://github.com/sethvargo/go-githubactions/issues/9#issuecomment-708914801, it looks like you might be making a follow-up PR? Do you want to add the comment about the panic in that?

Reply:
Sure, I'll document the panic in that PR!